src/frontend/app.ts

```diff
@@ -59,7 +59,7 @@ export class App {
   private selectedTab: Tab = Tab.ComponentTree;
   private subscription: Subscription;
   private tree: MutableTree;
-  private error: ApplicationError;
+  private error: ApplicationError = new ApplicationError(ApplicationErrorType.NotNgApp);
```

Comment:
@rajinder-yadav Can we change this so this error scenario is explicitly indicated from the backend (or more acurately, the `ng-validate` script)? As it is now we get a flash of this error message while Augury is starting up which I think it a little yucky.